src/main/java/net/dv8tion/jda/api/utils/MarkdownSanitizer.java

```diff
@@ -416,6 +416,8 @@ private void applyStrategy(int region, @Nonnull String seq, @Nonnull StringBuild
             throw new IllegalStateException("Found illegal region for strategy ESCAPE '" + region + "' with no known format token!");
         if (region == UNDERLINE)
             token = "_\\_"; // UNDERLINE needs special handling because the client thinks its ITALICS_U if you only escape once
+        else if (region == (BOLD | ITALICS_A))
+            token = "*\\*\\*"; // BOLD | ITALICS_A needs special handling because the client thinks its BOLD if you only escape once
```

Comment:
This probably also needs to be done for simple bold `\*\*` rather than `\**`

Reply:
Good catch, fixed - a few more tests were affected by this change, I tweaked them [in my latest commit](https://github.com/DV8FromTheWorld/JDA/pull/1087/commits/8f245bb7b3ef36f05c67fd67e7e4627ce7124560) to be more accurate on what's supposed to come out.